third_party/enum34.BUILD

```diff
@@ -0,0 +1,13 @@
+# Description:
```

Comment:
Why do we need this files?

Reply:
We needed the six.BUILD file but I'm now referencing the one packaged within io_abseil_py.